demos/src/simple.mustache

```diff
@@ -1,3 +1,37 @@
+<h1>Button toggle</h1>
+<div data-o-component="o-expander" class="o-expander items" data-o-expander-shrink-to="2" data-o-expander-count-selector="li">
+	<h2>Collapsing to number of items in a list</h2>
+	<ul class="o-expander__content">
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+	</ul>
+	<button class="o-expander__toggle o--if-js"></button>
+</div>
+
+<div data-o-component="o-expander" class="o-expander height" data-o-expander-shrink-to="height">
+	<h2>Collapsing to height of content (resize window to see toggle appear and disappear in a content-aware manner)</h2>
+	<div class="o-expander__content">
+		word word word word word word word word word word word word word word word word word word word word word word
+	</div>
+	<button class="o-expander__toggle o--if-js"></button>
```

Comment:
Should have some default text there though in case JS doesn't load. Otherwise it's an empty button/link...